Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Paged/PlaylistSwizzlerScript.js

```diff
@@ -4,12 +4,16 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 // Stub out the MediaSource API so video players do not attempt to use `blob` for streaming
-if (window.MediaSource || window.WebKitMediaSource || window.HTMLMediaElement && HTMLMediaElement.prototype.webkitSourceAddId) {
-  window.MediaSource = null;
-  window.WebKitMediaSource = null;
-  //HTMLMediaElement.prototype.webkitSourceAddId = null;
-  //window.SourceBuffer = null;
-  
+
+if (window.MediaSource || window.WebKitMediaSource || window.ManagedMediaSource || (window.HTMLMediaElement && HTMLMediaElement.prototype.webkitSourceAddId)) {
   delete window.MediaSource;
   delete window.WebKitMediaSource;
+  delete window.ManagedMediaSource;
+  
+//  window.MediaSource = undefined;
```

Comment:
Also could be a good idea to write description the reason of the change

Reply:
Added a comment